src/chart_types/partition_chart/layout/types/config_types.ts

```diff
@@ -62,6 +62,7 @@ export interface FillFontSizeRange {
   minFontSize: Pixels;
   maxFontSize: Pixels;
   idealFontSizeJump: Ratio;
+  monotonic: boolean;
```

Comment:
Add a tsdoc to describe the effect of this property `monotonic` 
```suggestion
  /** the fonts are scaled in.... **/
  monotonic: boolean;
```

Reply:
0629ff28970df61b527b674d1d3ea78bf6dded15

Reply:
note: it got renamed in a subsequent commit https://github.com/elastic/elastic-charts/commit/e166746701376d2810b075dc77d2304584e7ea01